### PR TITLE
Mirex2019

### PR DESCRIPTION
**Pull request by JamesOwers**
~Before I assumed that the files provided were pre normalization, but
examples provided looked to have already been softmaxed. The best
crossentropy is 0, the worst is infinite. I have added a tiny
constant to allow for the case where a model assigns 0 probability
then renormalized such that the 'crossentropy_score' ranges from
0 to 1 and the best value is 1.~

Scratch that, when you rescale (i.e. move from log space back to probability space), we end up just taking the average of the probability. I feel a bit of an idiot :joy:. This score is much better since the non informative model (predicts .5 for everything) gets a score of .5.

### CODE REVIEW

**Comment by BeritJanssen** [comment]
Ok! Thanks for checking!
